Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,22 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+op-version was introduced to ensure gluster bits running with different version of bits do not end up in a problem and backward compatibility issues can be tackled.
+
+After Gluster upgrade, it is advisable to have op-version updated.
+
+###Updating op-version
+
+Current op-version can be queried as below:
+
+    [root@~]#gluster volume get  <volume name> cluster.op-version
+
+op-version can be updated as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
```

Comment:
The Red Hat product documentation has this tabled [here](https://access.redhat.com/solutions/543123) . If I remember correctly, downstream and upstream op-versions may or may not be the same. Is there a similar op-version table upstream that a user can consume ? It'd be good to create one if it isn't there.

A developer can [find it in source](https://github.com/gluster/glusterfs/blob/master/libglusterfs/src/globals.h#L21-L73) but users won't.